osmosis-1/osmosis-1.assetlist.json

```diff
@@ -3391,6 +3391,65 @@
         "peg:collateralized"
       ]
     },
+    {
```

Comment:
Please do not add to the assetlist.json file directly. This is generated only

Reply:
thank you, removed